arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/InvalidReason.java

```diff
@@ -0,0 +1,6 @@
+package io.arex.inst.runtime.util;
+
+public enum InvalidReason {
+    FAST_REJECT,
```

Comment:
`FAST_REJECT` is a common state，which will be switched to first after queue overflow and service exceptions, so here should be `SERVICE_EXCEPTION` ?

Reply:
Duplicate with class `io.arex.foundation.model.DecelerateReasonEnum`